src/ui/components/Overlay/index.js

```diff
@@ -22,10 +25,15 @@ type UIStateType = {|
 
 const initialUIState: UIStateType = { visible: false };
 
+type keyDownType = {|
```

Comment:
My nit from the last iteration stands. I'm not sure you need to define a `keyDownType` when it's only used once, below. You could just define `InternalProps` to include 

`keydown: {| event: SyntheticEvent<any> | null |}`

Reply:
I missed that...yes I'll combine into InternalProps. That makes sense. thanks